helpers/generate_virify_samplesheet.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import csv
+import argparse
+from glob import glob
+
+import logging
+
+logging.basicConfig(level=logging.INFO)
+
+
+def chunk_list(data, chunk_size):
+    """Yield successive chunk_size-sized chunks from data."""
+    for i in range(0, len(data), chunk_size):
+        yield data[i:i + chunk_size]
+        
+
+def main():
+    parser = argparse.ArgumentParser(description='Generate a CSV samplesheet for VIRify using the output of genome '
+                                                 'catalogue pipeline.')
+    parser.add_argument('--species-catalogue-folder', required=True,
+                        help='Full path to the species_catalogue folder (inside the main output directory '
+                             'generated by genome catalogue pipeline')
+    parser.add_argument('--output', default='virify_samplesheet.csv', help='Output CSV filename (default: '
+                                                                           'virify_samplesheet.csv)')
+    parser.add_argument('--chunk-size', type=int, default=4000,
+                        help='Maximum number of records per CSV. Default: 4000')
+    args = parser.parse_args()
+
+    base_dir = args.species_catalogue_folder
+    output_csv_base = args.output
+    chunk_size = args.chunk_size
+
+    header = ['id', 'assembly', 'fastq_1', 'fastq_2', 'proteins']
```

Comment:
you can just leave 'id' and 'assembly' in samplesheet. No need to add the rest, those are optional, pipeline will still work